web/src/main/java/it/pagopa/selfcare/mscore/web/controller/InstitutionController.java

```diff
@@ -375,4 +375,32 @@ public ResponseEntity<Void> updateCreatedAt(@ApiParam("${swagger.mscore.institut
         return ResponseEntity.status(HttpStatus.OK).build();
     }
 
+    /**
+     * Retrieve institutions with productId onboarded
+     *
+     * @param productId     String
+     * @param page          Integer
+     * @param size          Integer
+     * @return List
+     * * Code: 200, Message: successful operation
+     * * Code: 404, Message: product not found
+     */
+    @ResponseStatus(HttpStatus.OK)
+    @ApiOperation(value = "${swagger.mscore.institutions.findFromProduct}", notes = "${swagger.mscore.institutions.findFromProduct}")
+    @GetMapping(value = "/products/{productId}")
```

Comment:
The current endpoint is ` /institutions/products/{productId}` because the product is a filter to retrieve institutions like this https://www.atatus.com/blog/rest-api-design-filtering-sorting-and-pagination/#filtering-methods